pkg/events/log_events.go

```diff
@@ -37,17 +45,31 @@ func (l loggingEventHandler) Handle(u *unstructured.Unstructured, e eventapi.Job
 		"gvk":        u.GroupVersionKind().String(),
 		"event_type": e.Event,
 	})
-	t, ok := e.EventData["task"]
-	if ok {
-		log = log.WithField("task", t)
+
+	if l.LogLevel == Nothing {
+		return
+	}
+
+	_, ok := e.EventData["task"]
+
+	// log only the following for the 'Tasks' LogLevel
+	if ok &&
+	   e.Event == EventPlaybookOnTaskStart &&
+	   e.EventData["task_action"] != TaskActionDebug &&
+	   e.EventData["task_action"] != TaskActionSetFact {
+		log.Infof("[playbook task]: %s", e.EventData["name"])
+		return
 	}
-	switch l.LogLevel {
-	case Everything:
+	if ok &&
+	   e.Event == EventRunnerOnOk &&
+       e.EventData["task_action"] == TaskActionDebug {
+		log.Infof("[playbook debug]: %v", e.EventData["task_args"])
+		return
+	}
```

Comment:
@johnkim76 I would like to see 1 more case here. 

if the event is "playbook_on_fail" do you think that we could log an error with the entire event? do people think that would be too unreadable? you could use the stdout in that case maybe?

Reply:
ahh missed failed caese.. will print easier-to-read failed cases